README.md

```diff
@@ -16,7 +18,10 @@ let conjugatedWord = koreanConjugator.conjugate(‘하다’, {
 ```
 
 # Tests
-`npm run test`
+```sh
+$ npm run test        # run single time
+$ npm run test:watch  # live re-run test for any change
```

Comment:
The reason why I changed this is that we need single run test script for CI. If the script keeps watching the change and run test, the CI is not done.